pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -650,6 +642,14 @@ func (r *ReconcileClusterDeployment) reconcile(request reconcile.Request, cd *hi
 		return reconcile.Result{}, nil
 	}
 
+	// Return early and stop processing if Agent install strategy is in play. The controllers that
+	// handle this portion of the API currently live in the assisted service repo, rather than hive.
+	// This will hopefully change in the future.
+	if cd.Spec.Provisioning != nil && cd.Spec.Provisioning.InstallStrategy != nil && cd.Spec.Provisioning.InstallStrategy.Agent != nil {
+		cdLog.Info("skipping processing of agent install strategy cluster")
+		return reconcile.Result{}, nil
+	}
+
```

Comment:
Hmm.. i think we might have to update the code for installed cluster code-path too.

L539-544
```go
		// delete failed provisions which are more than 7 days old
		existingProvisions, err := r.existingProvisions(cd, cdLog)
		if err != nil {
			return reconcile.Result{}, err
		}
		r.deleteOldFailedProvisions(existingProvisions, cdLog)
```

```

Reply:
Also clenaup for PVC ?
L547
```go
		r.cleanupInstallLogPVC(cd, cdLog)
```

Reply:
both these functions seem to be happy in case none exist from quick glance, so i don't think it's required per-se.